### DISCUSSION TITLE
graphql/codegen/plugins/presets/gql-tag-operations-preset

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#8969] [Docs Discussions]
# graphql/codegen/plugins/presets/gql-tag-operations-preset

GraphQL Code Generator Documentation

https://the-guild.dev/graphql/codegen/plugins/presets/gql-tag-operations-preset

**Comment by ivanakcheurov** [comment]
When using this plugin (`@graphql-codegen/gql-tag-operations-preset@2.0.0`), I get the following warning:
```
DEPRECATED: `gql-tag-operations-preset` is deprecated in favor of `client-preset`.
```
This warning hints it's better to move to `@graphql-codegen/client-preset` ([link](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client))
